libs/win32/ldns/ldns-lib/ldns-lib.2017.vcxproj

```diff
@@ -95,7 +95,7 @@
       <DisableSpecificWarnings>4013;4101;4996;4267;4244;%(DisableSpecificWarnings)</DisableSpecificWarnings>
     </ClCompile>
     <PreBuildEvent>
-      <Command>if not exist "$(ProjectDir)..\..\..\ldns\ldns\config.h" type "$(ProjectDir)\config.h" &gt; "$(ProjectDir)..\..\..\ldns\ldns\config.h"
+      <Command>type "$(ProjectDir)\config.h" &gt; "$(ProjectDir)..\..\..\ldns\ldns\config.h"
```

Comment:
Do we really need this?

Reply:
The short answer is YES.  However whether it is the best solution is very debatable.

Let me explain why this change is needed.

Currently the tar ball (http://files.freeswitch.org/downloads/libs/ldns-1.6.9-1-win.tar.gz) downloaded by the  download_LDNS.props includes **freeswitch\libs\ldns\ldns\config.h** (which is the file that needs to be changed (hence this PR)).  However since the tar ball cannot be changed then the file I changed was **freeswitch\libs\win32\ldns\ldns-lib\config.h**.  So somehow freeswitch\libs\win32\ldns\ldns-lib\config.h needs to replace freeswitch\libs\ldns\ldns\config.h in order for this fix to take effect.

**So the options are:**

1) Change freeswitch\libs\win32\ldns\ldns-lib\ldns-lib.2017.vcxproj to perform this task - which is what I have done already inside this PR.  This was based on what was already inside the project file.

or

2) Change the original tar ball contents with the fix inside freeswitch\libs\ldns\ldns\config.h - which you stated previously seemed like not an option.

or

3) Add a copy feature to download_LDNS.props to copy freeswitch\libs\win32\ldns\ldns-lib\config.h to overwrite freeswitch\libs\ldns\ldns\config.h during the first build of ldns.  Perhaps this could also include the other header files inside freeswitch\libs\win32\ldns\ldns-lib to be a more complete solution.  

Since you are one of the maintainers of the FreeSWITCH build system (especially the windows build) then I would really appreciate your ideas as to how to best make this work.

You are more than welcome to select an option from the above or suggest another option and I will follow your lead on this one.  The main aim is to get this live prior to the next release of FreeSWITCH.

Andrew

Reply:
We can change the tarball.

Reply:
Pushed an update.